eda/src/autogluon/eda/state.py

```diff
@@ -57,7 +57,7 @@ def at_least_one_key_must_be_present(self, state: AnalysisState, *keys) -> bool:
             True if at least one key from the `keys` list is present in the state
         """
         for k in keys:
-            if k in state:
+            if k in state and state[k] is not None:
```

Comment:
Doesn't matter but maybe this is simpler:
`if state.get(k, None) is not None`

Reply:
good suggestion; added